depthai_demo.py

```diff
@@ -172,51 +213,74 @@ def print_sys_info(info):
                 if enc_manager is not None:
                     enc_manager.parse_queues()
 
-                if sbb_out is not None and pv.has(Previews.depth.name):
+                if sbb_out is not None:
                     sbb = sbb_out.tryGet()
                     if sbb is not None:
                         sbb_rois = sbb.getConfigData()
-                    depth_frame = pv.get(Previews.depth.name)
-                    for roi_data in sbb_rois:
-                        roi = roi_data.roi.denormalize(depth_frame.shape[1], depth_frame.shape[0])
-                        top_left = roi.topLeft()
-                        bottom_right = roi.bottomRight()
-                        # Display SBB on the disparity map
-                        cv2.rectangle(pv.get("depth"), (int(top_left.x), int(top_left.y)), (int(bottom_right.x), int(bottom_right.y)), nn_manager.bbox_color[0], cv2.FONT_HERSHEY_SCRIPT_SIMPLEX)
+                    depth_frames = [pv.get(Previews.depth_raw.name), pv.get(Previews.depth.name)]
+                    for depth_frame in depth_frames:
+                        if depth_frame is None:
+                            continue
+
+                        for roi_data in sbb_rois:
+                            roi = roi_data.roi.denormalize(depth_frame.shape[1], depth_frame.shape[0])
+                            top_left = roi.topLeft()
+                            bottom_right = roi.bottomRight()
+                            # Display SBB on the disparity map
+                            cv2.rectangle(depth_frame, (int(top_left.x), int(top_left.y)), (int(bottom_right.x), int(bottom_right.y)), nn_manager.bbox_color[0], cv2.FONT_HERSHEY_SCRIPT_SIMPLEX)
             else:
                 read_correctly, host_frame = cap.read()
                 if not read_correctly:
                     break
 
-                scaled_frame = cv2.resize(host_frame, nn_manager.input_size)
-                frame_nn = dai.ImgFrame()
-                frame_nn.setSequenceNum(seq_num)
-                frame_nn.setType(dai.RawImgFrame.Type.BGR888p)
-                frame_nn.setWidth(nn_manager.input_size[0])
-                frame_nn.setHeight(nn_manager.input_size[1])
-                frame_nn.setData(to_planar(scaled_frame))
-                nn_in.send(frame_nn)
-                seq_num += 1
+                if nn_in is not None:
+                    scaled_frame = cv2.resize(host_frame, nn_manager.input_size)
+                    frame_nn = dai.ImgFrame()
+                    frame_nn.setSequenceNum(seq_num)
+                    frame_nn.setType(dai.RawImgFrame.Type.BGR888p)
```

Comment:
TODO handle input frame type when we will have blob parsing.
Also use `ImgFrame.Type.BGR888p` instead `RawImgFrame.Type.BGR888p`